cachelib/allocator/CacheAllocator.h

```diff
@@ -1153,6 +1172,44 @@ class CacheAllocator : public CacheBase {
     return stats;
   }
 
+  // returns the background mover stats
+  BackgroundMoverStats getBackgroundMoverStats(MoverDir direction) const {
```

Comment:
could these function be defined in Cache.h, and implemented in CacheAllocator.h like the other getStats functions?

Reply:
I think `getRebalancerStats` etc. are actually not implemented in the Cache.h and I just used the same approach.

Reply:
Actually, I think those functions should just be private. They are only used by `getStats` and `getGlobalStats`.

Reply:
ok.